cli/src/lint.ts

```diff
@@ -307,6 +307,16 @@ function lintPackageJson(
     }
     if (!Array.isArray(pkg.cloudflare.categories)) {
       problems.push('"cloudflare.categories" must be an array');
+    } else {
+      for (let i = 0; i < pkg.cloudflare.categories.length; i++) {
+        if (
+          !["starter", "storage", "ai"].includes(pkg.cloudflare.categories[i])
+        ) {
+          problems.push(
+            `"cloudflare.categories" lists "${pkg.cloudflare.categories[i]}", but can only include "starter", "storage", and "ai".`,
+          );
+        }
+      }
```

Comment:
Optional: You could simplify this a touch. I find `for` loops useful for transformations but sometimes a `forEach` can do the job simply enough.

```suggestion
      pkg.cloudflare.categories.forEach((cat) => {
        !["starter", "storage", "ai"].includes(cat) &&
          problems.push(
            `"cloudflare.categories" lists "${cat}", but can only include "starter", "storage", and "ai".`,
          );
      })
```

Reply:
Love it! Thanks :)